app/scripts/views/index.js

```diff
@@ -19,29 +20,20 @@ import ServiceMixin from './mixins/service-mixin';
 import Template from 'templates/index.mustache';
 
 class IndexView extends FormView {
-  constructor (options) {
-    super(options);
-
-    this.template = Template;
-  }
+  template = Template;
```

Comment:
What's this? Is it part of JS class syntax? I've not seen it before, looks magical!

Reply:
https://tc39.github.io/proposal-class-fields/, https://babeljs.io/docs/en/babel-plugin-transform-class-properties/

"class fields", a.k.a., "class properties", a.k.a., "instance properties". The important gotcha is they are added to the instance after the constructor is run rather than on the prototype. That has ramifications on the content server where things like "events" are expected to be on the prototype *before* the constructor is run.